java/java.lsp.server/test/unit/data/gradle-hello/app/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Ugly file, too ugly. Would it be to dare to ask you to create a Gradle project using the wizard/template in the IDE?

Reply:
ok, why not.

Reply:
project template used in 7cce3cf